test/spec/tasks/generate_spec_tests.ex

```diff
@@ -92,6 +92,7 @@ defmodule Mix.Tasks.GenerateSpecTests do
 
   defp generate_case(runner_module, testcase) do
     """
+      @tag :tmp_dir
```

Comment:
whats this for?

Reply:
The [`tmp_dir`](https://hexdocs.pm/ex_unit/main/ExUnit.Case.html#module-tmp-dir) tag tells `ExUnit` it should create a temporary directory for the test and adds its path to the context with `tmp_dir` key. The blob tests reuse the same blocks on different testcases, which causes the tests to fail if some of those testcases aren't supposed to have blobs. That's why we need to nuke the DB after finishing each test.

Reply:
nice!